### DISCUSSION TITLE
docker flutter_web_desk

### DISCUSSION THREAD

**Discussion by happytrudy** [rustdesk/rustdesk#6112] [General]
### Bug Description

Flutter_ Web_ Desk has not updated the Docker container for a long time, please synchronize and update it

The Windows client can log in, but the web client is also configured to not log in

### How to Reproduce

docker run -p 5000:5000 keyurbhole/flutter_web_desk

### Expected Behavior

no 

### Operating system(s) on local side and remote side

opencloud os 8

### RustDesk Version(s) on local side and remote side

1.2.2

### Screenshots

no

### Additional Context

centos8

**Comment by TBG-FR** [comment]
Did you work on that ? We should be able to build the web client from the current sources, but that requires a bit of effort... 

🔗  #2535 https://github.com/rustdesk/rustdesk/issues/4579#issuecomment-1581971865 #3742 etc.
